docs/diagrams/workflow.mmd

```diff
@@ -0,0 +1,27 @@
+graph TD
+
+subgraph dataops/sandbox/my-project
```

Comment:
It looks like this will fix it -- https://github.com/knsv/mermaid/issues/790#issuecomment-478860052

The online editor does the right thing. I think the mermaid cli tool and the online tool are out of sync.

![image](https://user-images.githubusercontent.com/3304040/56915727-19994c80-6a6c-11e9-831c-b35c30425d83.png)

Reply:
It looks better for me now too.